csaf/Cargo.toml

```diff
@@ -40,6 +40,8 @@ walker-common = { version = "0.6.0-alpha.7", path = "../common" }
 sequoia-openpgp = { version = "1", default-features = false }
 
 csaf = { version = "0.5", default-features = false, optional = true }
+clap = "4.4.8"
```

Comment:
We should avoid adding `clap` as a dependency to the library version of this crate. So we need to split up the "options" struct into "options" and "arguments" like with the other options as well.

Reply:
This still seems like an open issue: 

> We should avoid adding clap as a dependency to the library version of this crate. So we need to split up the "options" struct into "options" and "arguments" like with the other options as well.